### PR TITLE
First version of mongo service

### PR DESCRIPTION
**Pull request by Tonnu**

### CODE REVIEW

File: content/docs/services/mongodb.md [hunk 1]
```diff
@@ -3,3 +3,33 @@ tags: mongodb, services
 ---
 
 ## MongoDB
+You can run the standard Docker MongoDB image without requiring any extra parameters to be passed in.
```
Comment (mies) [inline content/docs/services/mongodb.md]: parameters -> environment variables?

File: content/docs/services/mongodb.md [hunk 2]
```diff
@@ -3,3 +3,33 @@ tags: mongodb, services
 ---
 
 ## MongoDB
+You can run the standard Docker MongoDB image without requiring any extra parameters to be passed in. 
+
+### Setup
+To use the basic [docker Image](https://registry.hub.docker.com/_/mongo/) in your build, you can use the following wercker.yml as a starting point:
```
Comment (mies) [inline content/docs/services/mongodb.md]: link to wercker.yml file

File: content/docs/services/mongodb.md [hunk 3]
```diff
@@ -3,3 +3,33 @@ tags: mongodb, services
...
+   steps:
+(...)
+```
+
+### Env vars
+Docker will expose the following env vars, which you can use to connect to mongo from within your application. 
+
+```sh
+MONGO_ENV_MONGO_MAJOR=3.0
+MONGO_PORT=tcp://172.17.0.20:27017
```
Comment (hatchan) [inline content/docs/services/mongodb.md]: We probably do not want to show the values of the env vars, as these will change over time, and builds

File: content/docs/services/mongodb.md [hunk 4]
```diff
@@ -3,3 +3,33 @@ tags: mongodb, services
...
+MONGO_ENV_MONGO_VERSION=3.0.2
+MONGO_PORT_27017_TCP=tcp://172.17.0.20:27017
+MONGO_PORT_27017_TCP_PROTO=tcp
+MONGO_PORT_27017_TCP_ADDR=172.17.0.20
+MONGO_NAME=/wercker-pipeline-554b80083ff4b3ee38001450/mongo
+MONGO_PORT_27017_TCP_PORT=27017
+```
+
+To learn about how to find out the IP adress of a service container you should use the environment variables exposed by Docker. You can read more about that in our article [_availble env vars_](http://devcenter.wercker.io/docs/services/available-env-vars.html).
```
Comment (hatchan) [inline content/docs/services/mongodb.md]: Can we get some line wrapping at 80 characters?

File: content/docs/services/mongodb.md [hunk 5]
```diff
@@ -3,3 +3,33 @@ tags: mongodb, services
...
 
 ## MongoDB
+You can run the standard Docker MongoDB image without requiring any extra parameters to be passed in. 
+
+### Setup
+To use the basic [docker Image](https://registry.hub.docker.com/_/mongo/) in your build, you can use the following wercker.yml as a starting point:
+
+```
+# wercker.yml
+box: yourbox
```
Comment (mies) [inline content/docs/services/mongodb.md]: use a "real" language box here like `google/golang` or something

**Comment by mies** [comment]
looks good, minor comments

File: content/docs/services/mongodb.md (hunk 1 above)
Comment (Tonnu) [inline content/docs/services/mongodb.md]: They're Docker parameters, but wercker env vars. Not sure which is better?

**Pull request merged**